helmfile.d/charts/gatekeeper/templates/policies/deployment-minimum-replicas.rego

```diff
@@ -1,20 +1,28 @@
 package k8sminimumreplicas
 
-        object_name = input.review.object.metadata.name
-        object_kind = input.review.kind.kind
+import future.keywords.in
 
-        violation[{"msg": msg}] {
-            spec := input.review.object.spec
-            not input_replica_limit(spec)
-            msg := sprintf("The provided number of replicas is low for %v: %v. Elastisys Compliant Kubernetes recommends a minimum of 2 replicas.", [object_kind, object_name])
-        }
+object_name = input.review.object.metadata.name
+object_kind = input.review.kind.kind
 
-        input_replica_limit(spec) {
-            provided := spec.replicas
-            min_replicas := input.parameters.min_replicas[_]
-            value_superior_min_replicas(min_replicas, provided)
-        }
+violation[{"msg": msg}] {
+    spec := input.review.object.spec
+    metadata := input.review.object.metadata
+    not input_replica_limit(spec)
+    not check_label(metadata)
+    msg := sprintf("The provided number of replicas is low for %v: %v. Elastisys Compliant Kubernetes recommends a minimum of 2 replicas.", [object_kind, object_name])
+}
 
-        value_superior_min_replicas(min_replicas, value) {
-            min_replicas > value
-        }
+input_replica_limit(spec) {
+    provided := spec.replicas
+    min_replicas := input.parameters.min_replicas
+    min_replicas <= provided
+}
+
+check_label(metadata) {
+    provided := metadata.labels
+    value := input.parameters.label
+
+    some key, _ in provided
+    key == value
```

Comment:
Does this mean the label has to be a specific value to have effect?

Reply:
It only check for the key part of the label, the value doesn't matter.